ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1760,6 +1765,79 @@ IopGetParentIdPrefix(PDEVICE_NODE DeviceNode,
     return Status;
 }
 
+static
+ULONG
+IopValidateID(IN PWCHAR Id,
+              IN SIZE_T MaxIdLen,
+              IN ULONG MaxSeparators,
+              IN BOOLEAN IsMultiSz)
+{
+   PWCHAR PtrSymbol;
```

Comment:
The function seems to have a strange 3-space indentation instead of the standard 4-space.

Reply:
I followed the format used in this module.

Reply:
Yeah but it is broken ^^

Reply:
I can change on 4-space indents. Even in the whole file (kernel files) :))

Reply:
But one function will have 4 spaces, and the rest with three )

Reply:
That's actually what we prefer: using the correct indentation / code-formatting for the function(s) you introduce, while leaving the rest untouched, so as to reduce the diff and not introduce unrelated changes.
Code formatting fixes are preferred to be committed separately in a different patch.

Reply:
When the mod changes (and the rules for formatting the code too), will it be necessary to change everything again? My opinion: if you have an old suit and you need to sew a lost button, then change everything to the same or sew exactly the same as everyone else. Otherwise your costume will look strange.

But I'm ready to obey the general rules. Tell me what to do and I will do it.